### PR TITLE
add styling for proportional legend, adjust pin colors

### PR DESCRIPTION
**Pull request by katelynnobrien**

### CODE REVIEW

File: demo-samples/totals.html [hunk 1]
```diff
@@ -6,34 +6,55 @@
...
-        }
-    </style>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
 </head>
 
 <body>
-    <div id="map"></div>
+    <div class="map-container">
+      <div id="map"></div>
+      <div class="legend">
```
Comment (jbranigan) [inline demo-samples/totals.html:15]: This legend is 🔥

File: demo-samples/styles.css [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  margin-right: 5px;
+  margin-top: -5px;
+  color: #ccc;
+  text-align: right;
+  width: 30px;
+  font-size: 10px;
+  line-height: 1;
+}
+
+.mapboxgl-popup {
```
Comment (jbranigan) [inline demo-samples/styles.css:105]: I think the `body.color` style may be affecting the popup font color:
image
If we add a color property here, will that override it?

**Review by jbranigan — COMMENTED** [review]
Exciting progress. I'm back to working on getting the pixel dimensions of the circles, and will let you know. After the one comment about the popup font color, this looks good to go.

File: demo-samples/styles.css (hunk 2 above)
Comment (katelynnobrien) [reply demo-samples/styles.css:105]: Nice catch - will fix!

**Pull request merged**